bashunit

```diff
@@ -34,6 +35,11 @@ while [[ $# -gt 0 ]]; do
       shift
       shift
       ;;
+    --verbose)
```

Comment:
Just to allow also the short option name

```suggestion
    --verbose|-v)
```

Reply:
Oh, right! I thought about it, but I forgot it. Thanks!